### DISCUSSION TITLE
Problem with app store (update)

### DISCUSSION THREAD

**Discussion by Dareq781** [elementary/appcenter#2236] [General]
After installing the OS I started updating. Everything seems to be OK, but after downloading the update I see information about the update in the appstore. When I try to click to install it I get the message: "Program update failure. This may be due to manually compiled software obtained from third parties or a package manager error. Manually refreshing the update may fix this problem". Expanding the details section shows the message: "Aborted due to failure (While applying additional data: The "apply extra" script failed, exit status 256". My question? Is this a bug? Or something else?
![Zrzut ekranu z 2024-12-15 19 35 39](https://github.com/user-attachments/assets/8a297838-6f86-4a51-bba7-3483f36729eb)

**Comment by teamcons** [comment]
In a terminal 
flatpak repair
Then
flatpak update

The issue is caused by some canonical hijinx im Ubuntu iirc

**Reply by Dareq781** [reply]
Hey. In my case, the flatpak update command was enough. I haven't had any problems since. All applications update fine. Thank you very much for your help. <3

**Reply by Dareq781** [reply]
The same update issue occurs again. The fltapak epair and flatpak update commands don't work.

**Reply by ryonakano** [reply]
Are you seeing completely the same error message and details as before?

**Reply by Dareq781** [reply]
I did some tests, updated flatpacks and it seems the problem isn't mine.

**Reply by Dareq781** [reply]
I had this problem once before when a new OS version was released and someone somewhere gave me a solution, but I can't find it.
